akka-docs-dev/rst/java/http/routing-dsl/overview.rst

```diff
@@ -68,4 +68,9 @@ to HTTP entities.
 
 Read more about :ref:`marshalling-java`.
 
+akka-http contains a testkit that simplifies testing routes. It allows to run test-requests against (sub-)routes
+quickly without running them over the network and helps with writing assertions on HTTP response properties.
+
+Read more about :ref:`testkit-java`.
```

Comment:
can we make it `http-testkit-java` right away? There exist more testkits (no conflict yet, but easier to call it more precisely right away)